### PR TITLE
Feature/client waldur.py

### PR DESCRIPTION
**Pull request by sndvsk**
@Fronox

### CODE REVIEW

File: src/client_waldur.py [hunk 1]
```diff
@@ -2,13 +2,69 @@
...
+
+
+# 4.
+def create_new_member():
+    # TODO
+    return None
+
+
+#create_project("1f8643e30e424c8cbfbb960301c20fb0", "test123", "1111")
+#create_order("73201e09c2724b4db6634fda8a3f7787", "8eea8a6825da45a6bc0342574cdb0a97", "f8afe3975cfa4dcc9a199b7ccf7c71bb")
```
Comment (sergei-zaiaev) [inline src/client_waldur.py]: Please, add a new line

File: src/client_waldur.py [hunk 2]
```diff
@@ -2,13 +2,69 @@
...
-# Waldur Project      -> http://localhost/api/projects/
-# Waldur Order        -> http://localhost/api/marketplace-offerings/
-# Add new members     -> http://localhost/api/user-invitations/
+# 1.Waldur Organization -> http://localhost/api/customers/
+# 2.Waldur Project      -> http://localhost/api/projects/
+# 3.Waldur Order        -> http://localhost/api/marketplace-offerings/
+# 4.Add new members     -> http://localhost/api/user-invitations/
 #   Cancel invitation -> http://localhost/api/user-invitations/<invitation_id>/cancel/
 
 # Remove organization is not working
```
Comment (sergei-zaiaev) [inline src/client_waldur.py:9]: What is this?

File: src/app.py [hunk 3]
```diff
@@ -3,6 +3,8 @@
 import json
 from client_eosc import get_events_from_eosc
 
+# from client_waldur import create_project
```
Comment (sergei-zaiaev) [inline src/app.py]: Do we need this comment?

File: src/client_waldur.py [hunk 4]
```diff
@@ -2,13 +2,69 @@
...
+    WALDUR_AUTH = {'username': USERNAME, 'password': PASSWORD}
+    r = requests.post('http://localhost/api-auth/password/', data=WALDUR_AUTH)
+    content = r.json()
+    return content['token']
+
+
+TOKEN = get_token()
+waldur_client = wc.WaldurClient(waldur_url, TOKEN)
+
+# waldur_client.create_project("1f8643e30e424c8cbfbb960301c20fb0", "test123", "1111")
```
Comment (sergei-zaiaev) [inline src/client_waldur.py]: Do we need these comment lines?

**Review by sergei-zaiaev — COMMENTED** [review]
After resolving these questions, mark the PR as non-draft. I will review and merge it myself.

File: src/client_waldur.py (hunk 4 above)
Comment (sndvsk) [reply src/client_waldur.py]: comments for test

**Pull request merged**